src/python/pants/backend/python/goals/package_python_binary.py

```diff
@@ -114,13 +119,13 @@ async def package_python_binary(
             )
         ),
     )
-    return BuiltPackage(two_step_pex.pex.digest, relpath=output_filename)
+    return BuiltPackage(two_step_pex.pex.digest, (BuiltPackageArtifact(output_filename),))
 
 
 @rule(level=LogLevel.DEBUG)
 async def create_python_binary(field_set: PythonBinaryFieldSet) -> CreatedBinary:
     pex = await Get(BuiltPackage, PackageFieldSet, field_set)
-    return CreatedBinary(pex.digest, pex.relpath)
+    return CreatedBinary(pex.digest, cast(str, pex.artifacts[0].relpath))
```

Comment:
How do we know there's only going to be one artifact here? EDIT: Ok, I can see it's because of the line just above that creates a 1-element tuple. I still think it could be useful to have an explicit method instead of using `[0]` and potentially missing some artifacts.

Reply:
This code is going away in 2.1.0. It only exists so that the deprecated `binary` goal still works.